### DISCUSSION TITLE
Realtime documentation

### DISCUSSION THREAD

**Discussion by boardmain** [supabase/supabase#18040] [Questions]
which is the correct documentation for realtime ?
this on github or supabase website

on supabase site i see

~~~
// Initialize the JS client
import { createClient } from '@supabase/supabase-js'
const supabase = createClient(SUPABASE_URL, SUPABASE_ANON_KEY)

// Create a function to handle inserts
const handleInserts = (payload) => {
  console.log('Change received!', payload)
}

// Listen to inserts
const { data: todos, error } = await supabase.from('todos').on('INSERT', handleInserts).subscribe()
~~~

and i have the error from is not a function....

**Comment by GaryAustin1** [comment]
I believe you are looking at the deprecated version 1 supabase-js...
![image](https://github.com/supabase/supabase/assets/54564956/a077479f-9dc4-400c-b944-d62769f9c5e3)

**Reply by GaryAustin1** [reply]
![image](https://github.com/supabase/supabase/assets/54564956/5ba70825-93b1-4b37-beb7-86d0782ca042)
version 2
